src/content/tools/devtools/extensions.md

```diff
@@ -5,22 +5,26 @@ description: Learn how to use and build DevTools extensions.
 
 ## What are DevTools extensions?
 
-[DevTools extensions](https://pub.dev/packages/devtools_extensions)
+[DevTools extensions][]
 are developer tools provided by third-party packages that are
 tightly integrated into the DevTools tooling suite.
 Extensions are distributed as part of a pub package,
 and they are dynamically loaded into DevTools when
 a user is debugging their app.
 
+[DevTools extensions]: {{site.pub-pkg}}/devtools_extensions
+
 ## Use a DevTools extension
 
-If your app depends on a package that provides a DevTools extension,
-the extension automatically shows up in a new tab when you open DevTools.
+If your app depends on a package that provides a
+DevTools extension, the extension automatically
+shows up in a new tab when you open DevTools.
```

Comment:
**nitpick:** Remove "automatically". If a user has to do something, it's manual. If a program or daemon does it, it's automatic. The adverb would be redundant. Also, condition before action, per GSG.
```suggestion
When you open DevTools and your app depends on
a package that provides a DevTools extension,
the extension displays in a new tab.
```